ios/Plugin/Banner/BannerExecutor.swift

```diff
@@ -146,9 +154,18 @@ class BannerExecutor: NSObject, GADBannerViewDelegate {
     /// Tells the delegate an ad request loaded an ad.
     public func bannerViewDidReceiveAd(_ bannerView: GADBannerView) {
         NSLog("bannerViewDidReceiveAd")
+
+        let width = bannerView.frame.width
+        var height = bannerView.frame.height
+        // For fixing issue 160 iOs banner overlapping content on iPhone with rounded corners                                
+        if let window = UIApplication.shared.keyWindow {
```

Comment:
same

Reply:
distante,  Thanks for your suggestion. I updated the code